### ISSUE TITLE
StrictMode error about leaked Closable object

### ISSUE THREAD

**Issue by ehehhh** [brianwernick/ExoMedia#278]
- [x] I have verified there are no duplicate active or recent bugs, questions, or requests

###### Include the following:
 - ExoMedia version: `3.0.1`
 - Device OS version: `6.0.1`
 - Devide Manufacturer: `LG`
 - Device Name: `Nexus 5`
 
###### Reproduction Steps
 1. Set release on detach from window to `false` 
 2. call `release()` 

###### Expected Result
No Strict Mode errors about leaked closable objects
###### Actual Result
A resource was acquired at attached stack trace but never released. See java.io.Closeable for information on avoiding resource leaks.
                                                               `java.lang.Throwable: Explicit termination method 'release' not called
                                                                   at dalvik.system.CloseGuard.open(CloseGuard.java:180)
                                                                   at android.view.Surface.setNativeObjectLocked(Surface.java:459)
                                                                   at android.view.Surface.<init>(Surface.java:145)
                                                                   at com.devbrackets.android.exomedia.core.video.ExoVideoView$EMExoVideoSurfaceTextureListener.onSurfaceTextureAvailable(ExoVideoView.java:276)`
###### Info
I am still getting this error with the latest version. It seems to me that if I have more than one EMVideoView, they share the same Surface or something?

Some background info on what I am doing:
I have implemented a ViewPager, where each page is a layout that contains EMVideoView and on each page change, I call `release()` on the "old" EMVideoView and prepare the "new" EMVideoView for playing and the error mostly appears right after the "new" player has started.

**Comment by brianwernick** [comment]
Stop opening duplicates!

If you read the documentation for `setReleaseOnDetachFromWindow(...)` it mentions you have to handle the releasing yourself (you aren't which is why you are getting the Strict Mode errrors)

**Comment by ehehhh** [comment]
I am sorry for creating a duplicate, I was just under the impression that the bug still exists in the 3.0.1 version and I created the other issue for the older version.

Anyway, I did read the documentation and all it says is to call `release()` manually to handle releasing correctly. As I said in the `Info` section of the issue, I am calling it, but I'm still experiencing this problem. Is it possible that the `Surface` isn't correctly released?
